.circleci/src/workflows/root.yml

```diff
@@ -0,0 +1,6 @@
+jobs:
+  - root-publish-github-release:
+      context: Audius Client
+      filters:
+        branches:
+          only: /(^release.*)$/
```

Comment:
do we want to restrict to `release-client`?

just guessing based on the 'Audius Client' context above

Reply:
Yes good catch! PR auto merged but I'll include it with my next one